nuxeo-runtime/nuxeo-connect-standalone/src/main/java/org/nuxeo/launcher/connect/ConnectRegistrationBroker.java

```diff
@@ -42,34 +45,79 @@
  */
 public class ConnectRegistrationBroker {
```

Comment:
Yes, indeed.

- For the new enum `TrialField` added recently, its `@version 9.2` has been added
- For the methods inside the enum `TrialField`, theirs `@version 9.2` have been added
- For other existing methods, I didn't change them.

For more detail, check PR https://github.com/nuxeo/nuxeo/pull/891 to see what have been added.